src/main/java/pages/GuestCheckoutPage.java

```diff
@@ -0,0 +1,84 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import static engine.drivers.WebDriverFactory.getWebDriverInstance;
+
+public class GuestCheckoutPage {
```

Comment:
Can you add Step class for that will use your methods to actually fill the page based on random customer (there is already method for getting him/her) 
?:)

Reply:
resolved